CHANGELOG.md

```diff
@@ -22,9 +22,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Parameters
 
-| Old parameter | New parameter |
-| ------------- | ------------- |
-|               |               |
+| Old parameter            | New parameter |
+| ------------------------ | ------------- |
+| skip_haplogrep3          | skip_tools    |
+| skip_fastp               |               |
+| skip_gens                |               |
+| skip_germlinecnvcaller   |               |
+| skip_peddy               |               |
+| skip_smncopynumbercaller |               |
+| skip_vcf2cytosure        |               |
```

Comment:
add skip_fastqc

Reply:
skip_fastqc doesn't exist actually. I just added that functionality to skip_tools 😅